pkg/controller/atlasproject/atlasproject_controller.go

```diff
@@ -242,7 +242,7 @@ func (r *AtlasProjectReconciler) ensureDeletionFinalizer(ctx context.Context, wo
 	if !project.GetDeletionTimestamp().IsZero() {
 		if customresource.HaveFinalizer(project, customresource.FinalizerLabel) {
 			if customresource.IsResourceProtected(project, r.ObjectDeletionProtection) {
-				log.Info("Not removing Atlas database user from Atlas as per configuration")
+				log.Info("Not removing Project from Atlas as per configuration")
```

Comment:
This change is already in the main, is your branch updated?

Reply:
will double check, this is probably because I wanted to revert o my prior changes before yesterdays fix.

Reply:
fixed